### PR TITLE
Abhi shandy feat bip47

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vhdykqkmsxbz5aclqnkiq7dwh4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10976)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/medx4s4spdem3l3uccnca64uom?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10977)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/opkm6a5d5eo4t4bkgerchxtmdi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10978)

File: blue_modules/storage-context.js [hunk 1]
```diff
@@ -120,6 +120,10 @@ export const BlueStorageProvider = ({ children }) => {
         setWalletTransactionUpdateStatus(WalletTransactionsStatus.ALL);
       }
       await BlueElectrum.waitTillConnected();
+      const paymentCodesStart = Date.now();
+      await fetchSenderPaymentCodes(lastSnappedTo);
+      const paymentCodesEnd = Date.now();
+      console.log('fetch payment codes took', (paymentCodesEnd - paymentCodesStart) / 1000, 'sec');
```
Comment (abhiShandy) [inline blue_modules/storage-context.js:126]: How long does it take?

**Review by abhiShandy — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: blue_modules/storage-context.js (hunk 1 above)
Comment (Overtorment) [reply blue_modules/storage-context.js:126]: 2-3 sec cold start to instantiate bip47 instance
(shouldbe faster on production builds)

+ ~500ms for a network fetch

**Pull request merged**